src/amo/components/Errors/ErrorComponent/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  className?: string,
+  code: number,
+  header: React.Element<any> | string,
+|};
+
+type InternalProps = {|
+  ...Props,
+|};
+
+export default class ErrorComponent extends React.Component<InternalProps> {
+  render() {
+    const { children, className, code, header } = this.props;
+
+    return (
+      <NestedStatus code={code || 400}>
```

Comment:
This will cause `code` to be `400` if `code` was passed as `0`, but I wonder if we need to enforce a set of valid values for `code`? E.g., someone could pass in `12345` as `code`, but I don't think that would be valid for `NestedStatus`, so I think we should have a check to ensure that `code` is an acceptable value, which at this point looks like `400, 401, 404, 500`.